.github/workflows/install.sh

```diff
@@ -4,6 +4,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+test -z ${FATS_DIR-} && FATS_DIR=`dirname "${BASH_SOURCE[0]}"`/../..
```

Comment:
rm

this script should never be invoked by anything other than the CI workflow.

Reply:
Why not? It's useful for testing. Seems harmless.